test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -93,6 +94,18 @@ def hook(context):
 
 @OnScenarioEnd
 def hook(context):
+    # search coredumps after every test scenario
+    # CI pipeline might fail although all tests are passing
+    coredumps = getCoredump()
+    if len(coredumps) > 0:
```

Comment:
Either you can check if the test is being run in CI, else we need to update documentation on how to get stacktrace while running the test locally.

Reply:
```suggestion
    if coredumps:
```